tests/services/test_path_migration.py

```
@@ -111,23 +109,22 @@ def test_migrate_with_failure(self, db_session, tmp_path, file_inventory_factory
         old_path.mkdir()
-        
+
         f1 = old_path / "file1.txt"
         f1.write_text("content1")
-        
+
         inv1 = file_inventory_factory(path=str(f1), storage_type=StorageType.COLD)
         path_id = inv1.path_id
-        
+
         def mock_move(src, dst):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `mock_move`

Add return type annotation: `NoReturn`

(ANN202)